### DISCUSSION TITLE
about rtsp streaming

### DISCUSSION THREAD

**Discussion by ZTG0421** [LizardByte/community#259] [Sunshine]
Sunshine has a good performance in rtsp streaming. I want to push the desktop to the live server through rtsp. Do you have any good suggestions?

**Comment by ReenigneArcher** [comment]
Use OBS.

**Reply by ZTG0421** [reply]
But OBS doesn't support rtsp XD...    OBS has a plugin but only can create a local rtsp server then push stream

**Comment by FengNewp** [comment]
I want to know how to use OBS to play the RTSP screen of Sunshine。

**Reply by ReenigneArcher** [reply]
This is a slightly different question than the original one asked here, I believe.

I guess you (or someone) would need to create a "Moonlight" plugin for OBS to be able to connect to Sunshine and receive the stream.

**Reply by FengNewp** [reply]
Hello! I don't want to use Moonlight to remotely use my computer, I want to be able to use a recording device similar to NVR to record operations when someone is remotely on my local area network computer. Of course, this is because I saw Sunshine using Rtsp, so I wanted to directly obtain the RTSP stream from Sunshine.

**Reply by ReenigneArcher** [reply]
You can't do that without moonlight. There's rtsp plugins for OBS I think.

**Reply by FengNewp** [reply]
OK
Thanks

[a comment by an automated account is not shown]
